### ISSUE TITLE
Emptycontent looks distorted on mobile

### ISSUE THREAD

**Issue by szaimen** [nextcloud/server#39398]
The subline text in emptycontent looks distored on mobile: 
![image](https://github.com/nextcloud/server/assets/42591237/8c4f9235-3613-4de8-bcb1-baed3568ebc5)

_Originally posted by @szaimen in https://github.com/nextcloud/server/issues/39229#issuecomment-1632357008_

**Comment by ezhil56x** [comment]
@szaimen I am interested to work on this. Can you give me an overview about this issue and steps to reproduce this?

**Comment by szaimen** [comment]
> Can you give me an overview about this issue and steps to reproduce this?

Just install nextcloud/server from master, open the files app, open the sharing options in the left sidebar and click on Shared. Then you should see the problem on a small screen.

**Comment by anniepauline** [comment]
Willing to work on it!

**Comment by jancborchardt** [comment]
@anniepauline yep, just go for it and let us know if you have any questions or need help. :)

**Comment by anjana-1311** [comment]
Where can I see the CSS files for the same issue? Please share the filepath of CSS.

**Comment by szaimen** [comment]
It probabl, needs to be adjusted here: https://github.com/nextcloud-libraries/nextcloud-vue/blob/master/src/components/NcEmptyContent/NcEmptyContent.vue

**Comment by anniepauline** [comment]
I am working on this issue, Is it already solved?

**Comment by anjana-1311** [comment]
After analyzing the vue code I'm still trying to understand where I can get CSS code for the same.

**Comment by szaimen** [comment]
> I am working on this issue, Is it already solved? Looks like a PR was made for the same issue [#39229 (comment)](https://github.com/nextcloud/server/pull/39229#issuecomment-1632357008)

No this is not a fiy for this specific issue, was just noticed there

**Comment by szaimen** [comment]
> After analyzing the vue code I'm still trying to understand where I can get CSS code for the same. @szaimen

As I said, it needs to be adjusted here: https://github.com/nextcloud-libraries/nextcloud-vue/blob/master/src/components/NcEmptyContent/NcEmptyContent.vue

Inside there is also the css code

**Comment by Plesin** [comment]
Hi @szaimen I've recently started selfhosting NextCloud and love it. Would like to contribute here and there if I have some time. Opened a tiny css fix for this issue. Please let me know if there are more UI issues I could look at.

**Comment by szaimen** [comment]
> Hi [@szaimen](https://github.com/szaimen) I've recently started selfhosting NextCloud and love it. Would like to contribute here and there if I have some time. Opened a tiny css fix for this issue. Please let me know if there are more UI issues I could look at.

Thanks for opening the PR! I requested some reviewers to review the PR :)
